sdk/template/.template.config/template.json

```diff
@@ -438,6 +436,14 @@
           }
         ]
       }
+    },
+    "ciFileName": {
```

Comment:
Why does it need to be a parameter? This just complicates using it when the target file name is known. Just use a rename directive: https://github.com/dotnet/templating/wiki/Reference-for-template.json#source-definition

This is a good reference to see other capabilities of templates. The fewer parameters the template has, the easier it is to use.

Reply:
Thanks @heaths. I updated to use 'rename directive' to rename the file name. Please help to review again. Thanks,